src/audio/module_adapter/iadk/iadk_module_adapter.cpp

```diff
@@ -12,6 +12,9 @@
 extern "C" {
 #endif
 
+#include <sof/audio/sink_api.h>
+#include <sof/audio/source_api.h>
```

Comment:
should put to above line 11? to make all include are together.

Reply:
done